packages/cxl-ui/scss/cxl-credential.scss

```diff
@@ -0,0 +1,132 @@
+:host {
+  display: flex;
+  margin: auto;
+  text-align: start;
+  color: initial;
+}
+
+.container {
+  position: relative;
+  z-index: 0;
+  box-sizing: border-box;
+  display: flex;
+  flex-direction: column;
+  justify-content: space-between;
+  width: 1464px;
+  height: 1120px;
+  padding: var(--lumo-space-xl);
+  overflow: hidden;
+  font-family: var(--lumo-font-family);
+  font-size: var(--lumo-font-size-m);
+  font-weight: 500;
+  color: var(--lumo-primary-contrast-color);
+  text-transform: uppercase;
+  background-color: #d61f2c;
```

Comment:
Did you check if we have this color in any --lumo or --cxl variable? It's better to stick with them if possible

Reply:
oops, slipped past prototyping